management/azure_mgmt_compute/spec/virtual_machines_spec.rb

```diff
@@ -135,7 +135,7 @@
 
     result = @resource_helper.network_client.network_interfaces.list_all
     result.each do |network_interface|
-      expect(network_interface.virtual_machine).to be_an_instance_of(MsRestAzure::SubResource)
+      expect(network_interface.virtual_machine).to be_an_instance_of(Azure::ARM::Network::Models::SubResource)
```

Comment:
Just curious: Is there any reason that you need full namespace to access SubResource?